src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_AMOUNT = 5;
```

Comment:
The constant `SIDE_LENGTH` is not self-explanatory. It's better to name it in a way that indicates its purpose. For example, `MAX_SIDE_LENGTH` or `MAX_FIGURE_SIZE` would give a clearer indication that it defines the maximum size for any side of a figure.